dietpi/dietpi-software

```diff
@@ -13041,6 +13265,24 @@ If no WireGuard (auto)start is included, but you require it, please do the follo
 			G_DIETPI-NOTIFY 2 'All microblog.pub data and settings are still retained.\n - You can remove these manually: rm -R /mnt/dietpi_userdata/microblog-pub'
 		fi
 
+		if To_Uninstall 210 # lemmy
+		then
+			Remove_Service lemmy-ui 1 1
+			Remove_Service lemmy 1 1
+			## Remove_Service postgres ??
+			G_EXEC userdel lemmy
```

Comment:
```suggestion
			Remove_Service lemmy-ui
			Remove_Service lemmy 1 1
```
The two `1` arguments tell the function to remove a user respectively a group with the same name ("lemmy"). So Removing the group is not needed anymore. Since there is no `lemmy-ui` user or group, no need to add the arguments there.

The PostgreSQL service should only be removed on PostgreSQL uninstall. The Lemmy database should be removed here, of course.